netiquette/README.md

```diff
@@ -0,0 +1,63 @@
+# BcnEng's slack netiquette
+
+Welcome to BcnEng's slack netiquette. If you don't agree with some guideline or whish to propose an improvement, feel free to open a pull request.
+
+## Definition
+
+From [wikipedia](https://en.wikipedia.org/wiki/Etiquette_in_technology#Netiquette):
+
+> Netiquette [...] is a set of social conventions that facilitate interaction over networks [...]
+
+## Conventions
+
+Here you will find a set of basic conventions that we find useful.
+
+### behaviour
+
+Be respectful with your colleagues and follow the [Code of Conduct](https://github.com/bcneng/bcneng/tree/master/coc) rules.
+
+### accounts
+
+As a people-first slack organization, company accounts are not allowed.
+
+### content
+
+Use threads when possible to keep the conversations organized.
+
+Pasting a random link sometimes does not add much value. Consider explaining why you did it.
+
+Consider removing the link previews.
+
+Use emoticons to give a quick feedback. Writing _haha_, _jaja_, _XDDD_ or _lol_ don't contribute much.
```

Comment:
`s/don't/doesn't`

Reply:
In fact, `do not` is better than using contractions in formal texts.